gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -996,6 +1011,8 @@ gst_tensor_filter_common_init_property (GstTensorFilterPrivate * priv)
   priv->prev_ts = GST_CLOCK_TIME_NONE;
   priv->throttling_delay = 0;
   priv->throttling_accum = 0;
+
+  priv->prop.shared_tensor_filter_key = g_strdup (DEFAULT_SHARED_KEY);
```

Comment:
default value should be null (to not set the shared model)

Reply:
I agree all the model representations have to be managed. The default value has been changed to NULL.